### PR TITLE
added model and greeting controller, and the gitignore

### PR DESCRIPTION
**Pull request by leoncalermo**
# Summary
- Added the book model
- the Greeting controller, and the gitignore
- it is another project built  wiht spring initializer, my bad

### CODE REVIEW

File: src/main/java/com/wolox/servingwebcontent/models/Book.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+    // Setter
+    public void setIsbn(String newIsbn) {
+        this.isbn = newIsbn;
+    }
+
+    public String getIsbn() {
+        return isbn;
+    }
+}
```
Comment (facucompe) [inline src/main/java/com/wolox/servingwebcontent/models/Book.java]:
```suggestion
}

```

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/training_java
+spring.datasource.username= training-java
+spring.datasource.password= training-java
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.hibernate.ddl-auto = update
+server.port = 8081
```
Comment (facucompe) [inline src/main/resources/application.properties]:
```suggestion
server.port = 8081

```

**Review by facucompe — APPROVED** [review]

File: src/main/java/com/wolox/servingwebcontent/models/Book.java [hunk 3]
```diff
@@ -0,0 +1,138 @@
+package com.wolox.servingwebcontent.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
```
Comment (facucompe) [inline src/main/java/com/wolox/servingwebcontent/models/Book.java:10]: JPA needs an empty constructor. It won't be able to instantiate this class after finding the information in the DB

**Review by facucompe — CHANGES_REQUESTED** [review]

File: src/main/java/com/wolox/servingwebcontent/models/Book.java (hunk 3 above)
Comment (leoncalermo) [reply src/main/java/com/wolox/servingwebcontent/models/Book.java:10]: so i should remove it right ?

**Review by facucompe — APPROVED** [review]

**Pull request merged**